MonoGame.Framework/Audio/Xact/AudioEngine.cs

```diff
@@ -381,6 +381,10 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        /// <summary>
+        /// Allows this object to attempt to free resources and perform other
+        /// cleanup operations before garbage collection reclaims the object.
+        /// </summary>
         ~AudioEngine()
```

Comment:
@dellis1972 @SimonDarksideJ should we be documenting private/protected methods that the developer using the public APIs won't ever call or use?

Reply:
It certainly doesn't hurt.  But as a general rule no.
We should approve this and then improve our guidance for future commits